### PR TITLE
Bug 1118422 - Fix SIM PIN error message to include how many tries are le...

### PR DESCRIPTION
**Pull request by tamarahills**
...ft.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=871f78876cf2)

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=873a54680b0a)

File: apps/communications/dialer/test/unit/mmi_test.js [hunk 1]
```diff
@@ -643,4 +643,42 @@ suite('dialer/mmi', function() {
...
+    test('Check error message for incorrect PIN is correct', function() {
+      var message = {
+        serviceCode: 'scPin',
+        statusMessage: 'emMmiErrorBadPin',
+        success: 'false',
+        additionalInformation: 2
+      };
+      MmiManager.notifyError(message);
+      sinon.assert.calledWithMatch(MockMmiUI.error,
+        'emMmiErrorBadPin\nemMmiErrorPinPukAttempts{\"n\":2}',
```
Comment (DouglasSherk) [inline apps/communications/dialer/test/unit/mmi_test.js]: We should use ```MockL10n``` instead of this direct string checking.

**Comment by try-server-hook** [comment]
[tamarahills tamarahills](https://github.com/tamarahills) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4517c68a582a)

**Pull request merged**